pyproject.toml

```diff
@@ -50,7 +50,6 @@ cachecontrol = { version = "^0.12.9", extras = ["filecache"] }
 cachy = "^0.3.0"
 cleo = "^1.0.0a5"
 crashtest = "^0.3.0"
-entrypoints = "^0.4"
```

Comment:
if you are removing this, remove `types-entrypoints` from dev section too

Reply:
Good catch.